src/pages/components/combobox.mdx

```diff
@@ -257,34 +257,34 @@ Nest a Combobox within a [Field](#field) component.
 <Component components={props.data.mdx.components} componentName="field" />
 
 A Field provides accessibility attributes to the child [Combobox](#combobox) field by associating it
-with the corresponding [Label](#label) and [Hint](#hint).
+with the corresponding [Label](#fieldlabel) and [Hint](#fieldhint).
 
 <PropSheet components={props.data.mdx.components} componentName="field" />
 
-### Hint
+#### Field.Hint
 
-<Component components={props.data.mdx.components} componentName="hint" />
+<Component components={props.data.mdx.subcomponents} componentName="hint" />
```

Comment:
To follow current dev patterns without errors, we need to update `Field`'s subcomponents `displayNames`.

For ex, `Hint` > `Field.Hint`:

https://github.com/zendeskgarden/react-components/blob/c28b41c6845d1500afed3507ae10e0d8cdf59137/packages/dropdowns/src/elements/combobox/Hint.tsx#L29